JenkinsConsoleUtility/JenkinsScripts/testInit.sh

```diff
@@ -27,6 +27,7 @@ ResetRepo (){
     git checkout master || git checkout -b master || CleanCurrentRepo
     git pull origin master
 
+    # Delete $GitDestBranch, reset it to master, prep for next build and fresh write
     if [ "$GitDestBranch"!="master" ]; then
         git fetch --progress origin
         git branch -D $GitDestBranch || true
```

Comment:
> || true [](start = 36, length = 8)

what is this for? "II true"

Reply:
I am telling Jenkins that if this line reports an error-code, that's ok, don't fail the Job.